apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -195,6 +195,8 @@ error.retry.code.invalid=Authentication failed. The code you entered is invalid
 authenticate=Continue
 no.valid.client.in.tenant=A valid client with the given client_id cannot be found in the tenant domain.
 not.authorized.to.use.requested.grant.type=The authenticated client is not authorized to use the requested grant type.
+authentication.flow.timeout=Something went wrong during the authentication flow
+authentication.flow.timeout.description=Seems like you have taken more time to complete the authentication. Try to sign in again. If the problem persists, please contact the administrator.
```

Comment:
Lets get this reviewed by the technical writers?
@nilminiwso2 @himeshsiriwardana @divyaamunugama